### ISSUE TITLE
Appium controller uses global version of appium when running from Node.js 

### ISSUE THREAD

**Issue by alex-prokopovich-cox** [aaronbriel/appium-controller#3]
@aaronbriel 
Appium controller started from CMD uses appium from dependencies, while started from Node.js uses global version of appium.

I think it should use local version, or at least have an option to select

**Comment by aaronbriel** [comment]
I would agree, although I'm not going to have time to do this for a while. If you'd like to do a PR that would work, otherwise this will have to wait for a month or so..
